react/forms/step2.md

```diff
@@ -0,0 +1,118 @@
+In this exercise, we'll be using a form demo to create a new form for a web application.
+
+The image below is what we'll end up with when we are done:
+
+<img src="forms/assets/step3.png" alt="Completed form with title field and email validation" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+
+## Task
+Katacoda is setting up a new React application for you. You'll be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
+
+1) Navigate to the `src` folder and open `App.js`
```

Comment:
Can you please add `{{open}}` at the end of this sentence? It's helpful for users to have this shorcut